.github/workflows/github-test.yaml

```diff
@@ -34,6 +35,25 @@ jobs:
           sparse-checkout: |
             .github
 
+      - if: ${{ matrix.folder == '.github' && matrix.os == 'ubuntu'}}
+        name: Lint workflows
+        run: |
+          set -x
+
+          # Ensure "shellcheck" is installed, since it's required for complete validation (pre-installed on GH agents)
+          command -v shellcheck >/dev/null || exit 1
+
+          # Download tool. Script SHA and tool version are pinned, but should be updated regularly.
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/2ab3a12c7848f6c15faca9a92612ef4261d0e370/scripts/download-actionlint.bash) 1.7.7
```

Comment:
I think we could just check this in with attribution. It's MIT licensed.

Reply:
What if we use the docker image instead?  How would we use a tool like this in an approved way?